app/graphql/types/stat_type.rb

```diff
@@ -0,0 +1,50 @@
+class Types::StatType < Types::BaseObject
```

Comment:
I don't think this is a good name, what if you renamed this to ProfileStats to be more clear? 🤔